scripts/publish-docker-images

```diff
@@ -127,12 +128,22 @@ push_multi-arch_manifest_docker_hub() {
 		rm multi-arch-temp.yaml
 	done
 
```

Comment:
nit: I spotted a few variables for paths used this function that are without `"$quotes"`, if you have cycles while updating for review feedback can you fix this in this function's scope?

Reply:
The variables here are still unquoted, are there pending changes?